install/kubernetes/cilium/charts/nodeinit/values.yaml

```diff
@@ -11,3 +11,6 @@ removeCbrBridge: false
 # Wait for the /var/run/azure-vnet.json file to be created before continuing the
 # script
 azure: false
+
+# Revert nodeinit changes via preStop container lifecycle hook
+preStopRevert: false
```

Comment:
maybe call this `revertReconfigureKubelet`

Reply:
Updated